playbooks/openstack/configuration.md

```diff
@@ -105,24 +104,23 @@ https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.examp
 
 ## OpenStack Cloud Provider Configuration
 
-The base OpenStack cloud provider configuration file provides limited parameters:
+Some features require you to configure the OpenStack cloud provider. For example, in
+`inventory/group_vars/OSEv3.yml`:
 
-```
-[Global]
-auth-url
-username
-password
-tenant-id / tenant-name
-domain-id (optional)
-domain-name (optional)
-region (optional)
+* `openshift_cloudprovider_kind`: openstack
+* `openshift_cloudprovider_openstack_auth_url`: "{{ lookup('env','OS_AUTH_URL') }}"
+* `openshift_cloudprovider_openstack_username`: "{{ lookup('env','OS_USERNAME') }}"
+* `openshift_cloudprovider_openstack_password`: "{{ lookup('env','OS_PASSWORD') }}"
+* `openshift_cloudprovider_openstack_tenant_name`: "{{ lookup('env','OS_PROJECT_NAME') }}"
+* `openshift_cloudprovider_openstack_domain_name`: "{{ lookup('env','OS_USER_DOMAIN_NAME') }}"
```

Comment:
I'll LGTM this, but what do you think of moving the backtick all the way to the end of the line?

I.e. instead of:

`openshift_cloudprovider_openstack_auth_url`: "{{ lookup('env','OS_AUTH_URL') }}"

It would be:

`openshift_cloudprovider_openstack_auth_url: "{{ lookup('env','OS_AUTH_URL') }}"`

I think it would be easier to read and copy/paste. But I'm not going to block the review on it.